certbot/docs/using.rst

```diff
@@ -802,64 +741,116 @@ configuration file we advise you to make a backup of the file beforehand and tes
 
 .. warning:: Manually modifying files under ``/etc/letsencrypt/renewal/`` can damage them if done improperly and we do not recommend doing so.
 
+.. _references:
 
-Automated Renewals
+Reference:
 ------------------
 
-Most Certbot installations come with automatic renewals preconfigured. This
-is done by means of a scheduled task which runs ``certbot renew`` periodically.
+.. _certbot_renew:
 
-If you are unsure whether you need to configure automated renewal:
+``certbot renew``
+~~~~~~~~~~~~~~~~~
+
+   This command attempts to renew any previously-obtained certificates that expire in less than 30 days. 
+   The same plugin and options that were used at the time the certificate was originally issued will be used for the renewal attempt, unless you specify other plugins or options. 
+   Unlike certonly, renew acts on multiple certificates and always takes into account whether each one is near expiry. 
+   Because of this, renew is suitable (and designed) for automated use, to allow your system to automatically renew each certificate when appropriate. 
+   Since renew only renews certificates that are near expiry it can be run as frequently as you want - since it will usually take no action.
+
+   .. note:: Options provided to ``certbot renew`` will apply to
+      *every* certificate for which renewal is attempted; for example,
+      ``certbot renew --rsa-key-size 4096`` would try to replace every
+      near-expiry certificate with an equivalent certificate using a 4096-bit
+      RSA public key. If a certificate is successfully renewed using
+      specified options, those options will be saved and used for future
+      renewals of that certificate.
+
+``-q``, ``--quiet``
+^^^^^^^^^^^^^^^^^^^
+   Silence all output except errors.
+
+``--force-renewal``
+^^^^^^^^^^^^^^^^^^^
+   For manually renewing all of your certificates
+
+   Causes the expiration time of the certificate(s) to be ignored when considering renewal, and attempts to
+   renew each and every installed certificate regardless of its age. (This form is not appropriate to run daily because each certificate will be
+   renewed every day, which will quickly run into the certificate authority rate limit.)
+
+Certbot Renew: Hooks
```

Comment:
I think the hooks section will eventually need a rewrite.

Some initial thoughts:

- Some of the advice, like running `certbot renew --pre-hook ...` is not up-to-date with the practices that we would suggest to users today. Especially for snap users, they cannot modify their scheduled task like that. Things should be done on a per-certificate basis using flags. (Or by using the `/etc/letsencrypt/renewal-hooks`, but those have a confusing caveat which makes me shy away from ever recommending that approach). `certbot reconfigure`, once merged, is relevant here too.
- We should cover the use of `--pre-hook`, `--post-hook`, `--deploy-hook`, on certificate creation, not at renewal time. That feels like the relevant time to learn about them. This could be a note in the "Getting certificates" section linking to a later section. 

I think it would also be helpful for the Renewal Hooks documentation to be more of a "How-to guide" and/or "Explanation" rather than a "Reference".